### ISSUE TITLE
No actual Window opens with Borg Application, unable to use at all

### ISSUE THREAD

**Issue by andrewhirsh** [Gautamji/borg#45]
I was using the app just fine a couple weeks ago but now Borg doesn't open anthing when I click on the application. I tried uninstalling and re-downloading
What should I do?

**Comment by Gautamji** [comment]
the settings file has most probably been corrupted. try the solution at #1 . delete the preferences.

**Comment by andrewhirsh** [comment]
Thanks Gautamji, How do I do this?
And I also want to say this is an incredible application you have here, and you have a bright future ahead of you

**Comment by Gautamji** [comment]
you can find the borg settings folder in 

Mac : ~/Library/Application Support/com.gautamji.borg/Local Store
For windows : C:\Users\{User name}\AppData\Roaming

first try removing the queue.dllist file and try
if it still does not work,
delete the settings folder and try again.

**Comment by andrewhirsh** [comment]
You're a genius!

**Comment by Prnixon** [comment]
MAC: i Can't find the file. After issue i deleted & reinstalled the app! help

**Comment by andrewhirsh** [comment]
Hi Guatam

I have tried  removing the queue.dllist file and deleting the settings
folder, I am still unable to download the course files. It is possible it
has to do with a problem with Adobe Air maybe? What do you think?
What do you recommend I do?
Thank you so much for your help
Best wishes,

Andrew Hirsh

**Comment by andrewhirsh** [comment]
To clarify the queue.dllist file keeps returning after I delete it from the
trash

**Comment by andrewhirsh** [comment]
Additionally now when I am trying to open the coursera website I get an
error message:
{"message":"too-many-requests","explanation":"You are temporarily blocked
from loading this URL"}
What does this mean?
Thank you for your help
Best wishes,

Andrew Hirsh

**Comment by Gautamji** [comment]
You get this error in the browser?

**Comment by Gautamji** [comment]
Which course are you getting problems in?

**Comment by andrewhirsh** [comment]
Yes, in the browser as well and I think for all courses though its possible
its just with coursera

**Comment by andrewhirsh** [comment]
Please help, any advice?
Thank you so much!

**Comment by andrewhirsh** [comment]
When I try to download youtube playlists it says "need more space to
download" but I should already have enough space on my computer, please
advise, thank you so much!
